ldscript.txt

```diff
@@ -9,7 +9,7 @@ SECTIONS
     {
         data/sound_data.o(ewram_data); 
         src/lib/m4a/m4a.o(ewram_data);
-        src/core.o(ewram_data);
+        src/malloc_ewram.o(.bss);
```

Comment:
I think it should still be marked as `EWRAM` for consistency ;) but I get this makes sense

Reply:
The build failed when I did that, it's what I wrote initially! :D

Reply:
But why lol, did you annotate with EWRAM_DATA

Reply:
I did.
And oh my god, I just found the fix for it.
You have to freaking assign it to 0 for the section attribute to work, for some reason.
https://github.com/SAT-R/sa2/pull/175/commits/68ebace2c93326173be9669b58166e8f201c0021#diff-adf4e26ad08a0add5f929909b0b974e522c2e1b5274bbff690737faf8826180eL9